stripe/src/main/java/com/stripe/android/net/StripeApiHandler.java

```diff
@@ -112,6 +112,7 @@ public static Source createSourceOnServer(
      * @throws APIConnectionException if there is a problem connecting to the Stripe API
      * @throws APIException for unknown Stripe API errors. These should be rare.
      */
+    @Deprecated
```

Comment:
People _can_ use StripeApiHandler instead of Stripe because the methods are public. That's why I've already moved it out of this package and made all methods package-private in the 5-0 :)